### PR TITLE
Rewords comment describing WRITETIME/TTL settings

### PR DESCRIPTION
**Pull request by arvy**
**What this PR does**:

Comment rewording based on feedback from a previous PR (the other PR was merged before I had a chance to make this change)

### CODE REVIEW

File: src/resources/cdm-detailed.properties [hunk 1]
```diff
@@ -102,16 +102,16 @@ spark.cdm.connect.target.password      cassandra
 #-----------------------------------------------------------------------------------------------------------
 spark.cdm.schema.origin.keyspaceTable                keyspace_name.table_name
 
-# Max TTL value of all non-PK columns will be used for insert on target
+# Max TTL value of all TTL-eligible non-PK columns will be used for insert on target
 #spark.cdm.schema.origin.column.ttl.automatic         true
 
-# Max TTL value of specified non-PK columns will be used for insert on target (overrides automatic setting)
```
Comment (msmygit) [inline src/resources/cdm-detailed.properties:108]: Curious, why are we removing `non-PK` verbiage from here as there won't be any writetime/ttl for primary key columns of a C* table?

Comment (arvy) [reply src/resources/cdm-detailed.properties:108]: The specified columns are implicitly non-PK columns - if they are not, validation will fail and feature will be disabled :)

Comment (arvy) [reply src/resources/cdm-detailed.properties:108]: I can add it back in (in parentheses)

Comment (msmygit) [reply src/resources/cdm-detailed.properties:108]: I think we should add it back as only non-PK columns has the writimetime/ttl values/properties.

Comment (msmygit) [reply src/resources/cdm-detailed.properties:108]: @arvy @arvydasj could you pls take care of this trivial doc update pr? Thank you in advance!

**Review by msmygit — CHANGES_REQUESTED** [review]
Requesting we apply the review comments and re-review to merge. Thanks.

**Comment by arvy** [comment]
I also removed the eclipse project files .classpath/.project - they are specified in the .gitignore list, which results in some funkiness

**Review by msmygit — APPROVED** [review]
LGTM 👍🏼

**Review by pravinbhat — APPROVED** [review]

**Pull request merged**